.gitignore

```diff
@@ -1,3 +1,4 @@
 out
 node_modules
 .vscode-test/
+analyzer.log
```

Comment:
This should go in the vscode ignore file too, so `vsce package` and `vsce publish` don't include it if it's on the disk.